src/integrationTest/java/uk/gov/hmcts/reform/sendletter/tasks/reports/DailyLetterUploadSummaryReportTest.java

```diff
@@ -21,7 +21,7 @@
     private JavaMailSender mailSender;
 
     @Test
-    public void should_attempt_to_send_report_when_recipients_list_is_present() {
+    void should_attempt_to_send_report_when_recipients_list_is_present() {
```

Comment:
why this change, tests in classes below still have `public`

Reply:
below where?

Reply:
in this PR, `StaleLetterControllerTest.java`

Reply:
The test was created by another team member with their preferences instead of junit5 guidelines. The reason this PR includes changes in `StaleLetterControllerTest` is purely to fix after transitioning from spring boot 2.1 to 2.2. Since (have no idea how and when) there were some junit4 leftovers I have applied junit5 guidelines as per test "fix"/upgrade. Once again, this was never picked up because we transitioned to junit5 engine hence never failed. Spring boot 2.2 also transitioned to junit5 but with vintage support - that is ability to run junit4 tests. We would not have such problem if we removed junit4 dependency completely but it is not possible since testcontainers have hard dependency on it